libtransmission/rpc-server.cc

```diff
@@ -652,7 +652,11 @@ void start_server(tr_rpc_server* server)
 
     auto* const base = server->session->event_base();
     auto* const httpd = evhttp_new(base);
-
+    if (!httpd) 
+    {
+        tr_logAddError(fmt::format(_("Couldn't instantiate httpd for '{server}'"), fmt::arg("server", server)));
```

Comment:
build error:
>"Formatting of non-void pointers is disallowed."

sorry, it's my fault. Try this:
```suggestion
tr_logAddError(fmt::format(_("Couldn't instantiate httpd for '{}'"), static_cast<void*>(server)));
```